From 496a70d54a15b00a1228c0ffad926de9573c04a1 Mon Sep 17 00:00:00 2001
From: Lianwei Wang <a22439@motorola.com>
Date: Fri, 19 Feb 2016 10:04:10 -0800
Subject: [PATCH 684/959] IKSWM-21322 config: 8952: enable mmc deferred resume

Enable mmc deferred resume to avoid unnecessary resume of sd card.
This can also save the system resume time.

Change-Id: Ibf83ac8d95c3c35901d63bb1e25d557e7c142ee1
Signed-off-by: Lianwei Wang <a22439@motorola.com>
Reviewed-on: http://gerrit.mot.com/823683
SME-Granted: SME Approvals Granted
SLTApproved: Slta Waiver <sltawvr@motorola.com>
Tested-by: Jira Key <jirakey@motorola.com>
Reviewed-by: Zhi-Ming Yuan <a14194@motorola.com>
Reviewed-by: Sheng-Zhe Zhao <a18689@motorola.com>
Submit-Approved: Jira Key <jirakey@motorola.com>
---
 arch/arm/configs/ext_config/moto-msm8916.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/ext_config/moto-msm8916.config b/arch/arm/configs/ext_config/moto-msm8916.config
index 22ae77c1a57..3d4a3fa7eb9 100644
--- a/arch/arm/configs/ext_config/moto-msm8916.config
+++ b/arch/arm/configs/ext_config/moto-msm8916.config
@@ -68,3 +68,4 @@ CONFIG_F2FS_FS=y
 CONFIG_F2FS_FS_SECURITY=y
 CONFIG_F2FS_FS_XATTR=y
 CONFIG_QPNP_USER_RESET=y
+CONFIG_MMC_BLOCK_DEFERRED_RESUME=y
-- 
2.11.0

